### PR TITLE
Issue a warning when add-on is already signed. (bug 1070188)

### PR DESCRIPTION
**Pull request by magopian**
Fixes [bug 1070188](https://bugzilla.mozilla.org/show_bug.cgi?id=1070188)

### CODE REVIEW

**Comment by magopian** [comment]
@mstriemer @clouserw r?

This supersedes #261

File: validator/testcases/content.py [hunk 1]
```diff
@@ -258,6 +258,23 @@ def test_packed_scripts(err, xpi_package):
             err.pop_state()
 
 
+@decorator.register_test(tier=2)
+def test_signed_xpi(err, xpi_package):
+    """Checks if XPI is signed."""
+    search = set([
+        'META-INF/manifest.mf',
+        'META-INF/zigbert.rsa',
+        'META-INF/zigbert.sf'])
```
Comment (mstriemer) [inline validator/testcases/content.py]: Is this the only change?

**Comment by mstriemer** [comment]
Only seeing the one change, lgtm. r+

File: validator/testcases/content.py (hunk 1 above)
Comment (magopian) [inline validator/testcases/content.py]: This line is correctly indented, and the `tier` in the decorator was changed from `1` to `2` on your advice.

**Comment by magopian** [comment]
Thanks for the review!

**Pull request merged**